phoible/views.py

```diff
@@ -1 +1,20 @@
-#
+from pyramid.view import view_config
+
+faq_page = None
+conventions_page = None
+with open('phoible/static/faq_with_indexes.html', 'r') as file:
```

Comment:
This assumes a certain working directory at the moment the module is imported. Better use paths relative to `__file__`, i.e. package-relative paths.

I also think reading these files in the global scope, i.e. when importing the module, feels a bit hacky. It means the app won't startup when a file read fails - when it actually only affects a single view. So it's better to read the file for each single request, I'd say. The preformance penalty of this should be negligible.

Reply:
Thank you for the suggestions! I will apply and fix it.